eva_cttv_pipeline/trait_mapping/main.py

```diff
@@ -0,0 +1,83 @@
+from collections import Counter
+import csv
+import progressbar
+
+from eva_cttv_pipeline.trait_mapping.output import output_trait
+from eva_cttv_pipeline.trait_mapping.oxo import get_oxo_results
+from eva_cttv_pipeline.trait_mapping.oxo import uris_to_oxo_format
+from eva_cttv_pipeline.trait_mapping.trait import Trait
+from eva_cttv_pipeline.trait_mapping.trait_names_parsing import parse_trait_names
+from eva_cttv_pipeline.trait_mapping.zooma import get_zooma_results
+
+
+def get_uris_for_oxo(zooma_result_list: list) -> set:
+    """
+    For a list of Zooma mappings return a list of uris for the mappings in that list with a high
+    confidence.
+
+    :param zooma_result_list: List with elements of class ZoomaResult
+    :return: set of uris from high confidence Zooma mappings, for which to query OxO
+    """
+    uri_set = set()
+    for mapping in zooma_result_list:
+        # Only use high confidence Zooma mappings for querying OxO
+        if mapping.confidence.lower() != "high":
+            continue
+        uri_set.update([entry.uri for entry in mapping.mapping_list])
```

Comment:
You're right, I think I might have been expecting to need more code after the uri_set update